contracts/mock/rysk/MockRyskLiquidityPool.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import "hardhat/console.sol";
+
+contract MockRyskLiquidityPool is ERC20 {
+    IERC20 immutable underlying;
+    uint256 public epoch;
+    mapping(uint256 => uint256) public epochPricePerShare;
+    mapping(address => DepositReceipt) public depositReceipts;
+    mapping(address => WithdrawalReceipt) public withdrawalReceipts;
+
+    struct DepositReceipt {
```

Comment:
Why not use `IRyskLiquidityPool` in here? You could avoid declaring these structs twice

Reply:
[fixed](https://github.com/lindy-labs/sc_solidity-contracts/pull/152/commits/3ea1cd4af18aa7f1f8b30b9a6724955439c3f857)